src/styles/blocks/product.css

```diff
@@ -0,0 +1,32 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  text-align: left;
+}
+
```

Comment:
for this file name should be same as block name `product-card.css`
or class name should be `product`